types/emoney_inflation.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import (
+	"time"
+
+	inflationtypes "github.com/e-money/em-ledger/x/inflation/types"
+)
+
+type EmoneyInflation struct {
```

Comment:
I think this should be `EMoneyInflation` if I'm not mistaken. There is a typo here.

Reply:
thanks! fixed and pushed. btw I changed the file name emoney_inflation.sql to emoney.sql as there are more customized modules coming, I will group the SQL CMDs in this file.